contracts/contracts/coordination/subscription/BqETHSubscription.sol

```diff
@@ -21,17 +21,19 @@ contract BqETHSubscription is EncryptorSlotsSubscription, Initializable, Ownable
         uint128 encryptorSlots; // pre-paid encryptor slots for the billing period
     }
 
-    IERC20 public immutable feeToken;
-
     uint32 public constant INACTIVE_RITUAL_ID = type(uint32).max;
+    uint192 public constant INCREASE_BASE = 10000;
```

Comment:
Any particular reason this is uint192 and not uint256?

Reply:
my bad, when copied - just missed that type is not 256, I'll change it

Reply:
✔️